README.md

```diff
@@ -184,3 +184,7 @@ Keras was initially developed as part of the research effort of project ONEIROS
 >_"Oneiroi are beyond our unravelling --who can be sure what tale they tell? Not all that men look for comes to pass. Two gates there are that give passage to fleeting Oneiroi; one is made of horn, one of ivory. The Oneiroi that pass through sawn ivory are deceitful, bearing a message that will not be fulfilled; those that come out through polished horn have truth behind them, to be accomplished for men who see them."_ Homer, Odyssey 19. 562 ff (Shewring translation).
 
 ------------------
+
+## License
+
+Read the [license](https://github.com/fchollet/keras/blob/master/LICENSE) for more information
```

Comment:
This part isn't necessary, the license is already mentioned prominently in the readme.